src/components/Navbar.js

```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom';
+
+const Navbar = () => { //eslint-disable-line
+  return (
+    <div className="navBar">
+      <h1 className="logo">Bookstore CMS</h1>
+      <ul className="navItems">
+        <li className="navItem"><Link to="/Book">Book</Link></li>
```

Comment:
- [ ] Same here, Please make the `Book` component the main route (path). It will be like that:
```javascript
<li className="navItem"><Link to="/">Book</Link></li>
```